### PR TITLE
Remove MGImageUtilities pod from our project.

### PR DESCRIPTION
**Pull request by SergioEstevao**
Fixes dependencies of outside pods:)

Check the comments section on the reader and the notifications section to see if all gravatar images are being displayed properly.

Needs review: @jleandroperez and @aerych I think the Reader and Notifications where the main users of these methods do you mind to have a look?

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Comments/WPStyleGuide+Comments.swift [hunk 1]
```diff
@@ -66,7 +65,7 @@ extension WPStyleGuide
         
         public static func timestampImage(isApproved approved: Bool) -> UIImage {
             let timestampImage = UIImage(named: "reader-postaction-time")!
-            return approved ? timestampImage : timestampImage.imageTintedWithColor(WPStyleGuide.alertYellowDark())
+            return approved ? timestampImage : timestampImage.imageWithRenderingMode(UIImageRenderingMode.AlwaysTemplate)
```
Comment (jleandroperez) [inline WordPress/Classes/ViewRelated/Comments/WPStyleGuide+Comments.swift]: No need to specify `UIImageRenderingMode.`, you can just do:

`timestampImage.imageWithRenderingMode(.AlwaysTemplate)`

File: WordPress/Classes/ViewRelated/Me/AboutViewController.swift [hunk 2]
```diff
@@ -26,8 +26,9 @@ public class AboutViewController : UITableViewController
     private func setupTableView() {
         // Load and Tint the Logo
         let color                   = WPStyleGuide.wordPressBlue()
-        let tintedImage             = UIImage(named: "icon-wp")?.imageTintedWithColor(color)
+        let tintedImage             = UIImage(named: "icon-wp")?.imageWithRenderingMode(UIImageRenderingMode.AlwaysTemplate)
```
Comment (jleandroperez) [inline WordPress/Classes/ViewRelated/Me/AboutViewController.swift]: Same as above, you can safely remove the `UIImageRenderingMode` prefix

File: WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingStreamsViewController.swift [hunk 3]
```diff
@@ -170,9 +170,8 @@ public class NotificationSettingStreamsViewController : UITableViewController
         case .Device:
             imageName = "notifications-phone"
         }
-        
-        let tintColor = WPStyleGuide.greyLighten10()
-        return UIImage(named: imageName)?.imageTintedWithColor(tintColor)
+
+        return UIImage(named: imageName)?.imageWithRenderingMode(UIImageRenderingMode.AlwaysTemplate)
```
Comment (jleandroperez) [inline WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingStreamsViewController.swift]: (Sorry about the noise!) `UIImageRenderingMode` prefix

**Comment by jleandroperez** [comment]
@SergioEstevao added a minor Swift comment.

:shipit: on my side sir!

**Comment by aerych** [comment]
Looks good to me, apart from @jleandroperez's comments.   
:shipit: :)

**Pull request merged**